spec/integration/api/conversation_visibilities_spec.rb

```diff
@@ -0,0 +1,86 @@
+require "spec_helper"
+
+
+describe Api::V0::ConversationVisibilitiesController do
+  let(:auth) { FactoryGirl.create(:auth_with_read_and_write) }
+  let(:auth_participant) { FactoryGirl.create(:auth_with_read_and_write) }
+  let!(:access_token) { auth.create_access_token.to_s }
+  let!(:access_token_participant) { auth_participant.create_access_token.to_s }
+
+  before do
+    auth.user.seed_aspects
+    auth.user.share_with auth_participant.user.person, auth.user.aspects[1]
+    auth_participant.user.share_with(
+        auth.user.person, auth_participant.user.aspects[0]
+    )
+
+    @conversation = {
+      author_id: auth.user.id,
+      subject: "new conversation",
+      text: "first message",
+      messages: [{
+       author: auth.user, 
+       text: "first message"
+      }],
+      recipients: [auth_participant.user.person.id],
+      access_token: access_token
+    }
+  end
+
+  describe "#destroy " do
+    before do
+      conversation = post api_v0_conversations_path(), @conversation
```

Comment:
- Useless assignment to variable - `conversation`.
- Do not use parentheses for method calls with no arguments.